unbound/datadog_checks/unbound/data/conf.yaml.example

```diff
@@ -0,0 +1,43 @@
+init_config:
+
+instances:
+
+  -
+    ## @param unbound_control - string - optional
+    ## unbound-control binary to execute to gather stats, defaults to unbound-control
+    #
+    # unbound_control: /path/to/unbound-control
+
+    ## @param use_sudo - boolean - optional - default: false
+    ## If your environment requires sudo, please add the following line:
+    ##
+    ## dd-agent ALL=(ALL) NOPASSWD:/usr/sbin/unbound-control
+    ##
+    ## (or a similar one with the correct path) to your sudoers file, and
+    ## uncomment the option below:
+    #
+    # use_sudo: true
+
+    ## @param stats_command - string - optional
+    ## unbound-control command to execute to gather stats, defaults to stats
```

Comment:
I'm pondering changing the default here to stats_noreset.  See https://nlnetlabs.nl/documentation/unbound/unbound-control/ for a description of stats and stats_noreset.  Trying to think of when we'd ever want to reset the counters.  Thoughts?

Reply:
> Trying to think of when we'd ever want to reset the counters. Thoughts?
Your call I think... I am not familiar with what the internal counters do, do you have an example in mind we could look at? To be fair you can leave it to `stats` by default. Can be reconsidered later one if need be.

Reply:
unbound's counters are the source of the metrics reported here.  https://nlnetlabs.nl/documentation/unbound/howto-statistics/ describes setting `statistics-cumulative` to `no` which is what makes it OK to use the stats command here.

So, what's the easier thing?  To leave stats here, and require people to set `statistics-cumulative`, or use stats_noreset here and then maybe it doesn't matter what `statistics-cumulative` is set to.  I can't think of a reason why resetting the counters is helpful, except that eventually I suppose they'll wrap.

Reply:
Let's go with `stats` by default. No change needed then.

Reply:
Sure.